dojo-starter/src/systems/actions.cairo

```diff
@@ -3,7 +3,8 @@ use dojo_starter::models::game::Game;
 // define the interface
 #[dojo::interface]
 trait IActions {
-    fn start_game() -> Game;
+    fn start_game(world:IWorldDispatcher) -> Game;
```

Comment:
you don't have to add the `world:IWorldDispatcher` but there is no harm

Reply:
okay, keeping it for now.